qa/sys/debian/user_bootstrap.sh

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env bash
 
 VERSION=`cat /vagrant/config/platforms.json | grep  latest | cut -d":" -f2 | sed 's/["\|,| ]//g'`
-LOGSTASH_FILENAME="logstash-${VERSION}_all.deb"
+LOGSTASH_FILENAME="logstash-${VERSION}.deb"
```

Comment:
yeah, before alpha3 we had arch in the package name, not anymore.

On Tue, Jun 14, 2016 at 10:49 AM João Duarte <notifications@github.com>
wrote:

> In qa/sys/debian/user_bootstrap.sh
> <https://github.com/elastic/logstash/pull/5486#discussion_r66931109>:
>
> > @@ -1,6 +1,5 @@
> >  #!/usr/bin/env bash
> >
> >  VERSION=`cat /vagrant/config/platforms.json | grep  latest | cut -d":" -f2 | sed 's/["\|,| ]//g'`
> > -LOGSTASH_FILENAME="logstash-${VERSION}_all.deb"
> > +LOGSTASH_FILENAME="logstash-${VERSION}.deb"
>
> why is this change in this PR? I'm not familiar with the naming
> requirements, are we removing architectures from package filenames?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/elastic/logstash/pull/5486/files/6e8a857c5e353baec1a214feabcfd02feee8e3fe#r66931109>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAELvKTBS4vQRAMsSOsyKxalkLMfJvRnks5qLmsjgaJpZM4I1E8l>
> .
>